tests/CompileAndExec.integ_test.cpp

```
@@ -0,0 +1,817 @@
+
+namespace {
+	std::string valueToString(const AsTL::StackValue &val) {
+		using namespace AsTL;
+
+		std::string retVal = "???";
+
+		std::visit(OverloadedVisit {
+			[&retVal](IDX arg)					{ retVal = std::to_string(arg); },
+			[&retVal](I32 arg)					{ retVal = std::to_string(arg); },
+			[&retVal](F64 arg)					{ retVal = std::to_string(arg); },
+			[&retVal](BOOL arg)					{ retVal = std::to_string(arg); },
```

Clang (14.0.6)

[warning] implicit conversion bool -> 'int'

(readability-implicit-bool-conversion)